### PR TITLE
Mention labelled parameters in Values and Functions

### PR DESCRIPTION
**Pull request by PoorlyDefinedBehaviour**
Part of https://github.com/ocaml/ocaml.org/issues/1944

### CODE REVIEW

**Review by christinerose — APPROVED** [review]
LGTM on my end. @cuihtlauac?

**Comment by cuihtlauac** [comment]
> LGTM on my end. @cuihtlauac?

I want to change that example. @PoorlyDefinedBehaviour can you give me access rights to your fork? I'll send you a commit

**Comment by PoorlyDefinedBehaviour** [comment]
> > LGTM on my end. @cuihtlauac?
> 
> I want to change that example. @PoorlyDefinedBehaviour can you give me access rights to your fork? I'll send you a commit

I thought you already had access to the branch
image

File: data/tutorials/language/0it_00_values_functions.md [hunk 1]
```diff
@@ -297,6 +297,15 @@ The `max` function returns the largest of its two arguments, which are:
...
 
+```ocaml
+# let square ~x = x * x;;
+val square : x:int -> int
+
+# square ~x:3;;
+- : int = 9
+```
+
+Functions with named parameters can be applied in the same way as functions without named parameters with the difference that the argument should be named. Note the `~x` when `square` is applied to `3`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
````suggestion
```ocaml
# String.starts_with ~prefix:"state" "stateless";;
- : bool = true
```

Some functions, such as `String.starts_with` have labelled parameters. Labels are useful when a function has several parameters of the same type; naming arguments allows to guess their purpose. Above, `~prefix:"state"` indicates `"less"` is passed as labelled argument `suffix`.

```ocaml
# let cat ?(init="") u = List.fold_left ( ^ ) init u;;
val cat : ?init:string -> string list -> string = <fun>

# cat ["fri"; "sat"; "sun"];;
- : string = "frisatsun"

# cat ["fri"; "sat"; "sun"] ~init:"thr";;
- : string = "thrfrisatsun"
```

Labelled parameters can have default values. This allows calling the function without specifying some arguments. Optional parameters when a function has many parameters or parameters which define some non nominal behaviour. Optional parameters are defined using a question mark `?` and passed using a tilde `~`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
````

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: data/tutorials/language/0it_00_values_functions.md [hunk 2]
```diff
@@ -297,6 +297,25 @@ The `max` function returns the largest of its two arguments, which are:
 - `42`, the result of `21 * 2`
 - `713`, the result of `int_of_string "713"`
 
+```ocaml
+# String.starts_with ~prefix:"state" "stateless";;
+- : bool = true
+```
+
+Some functions, such as `String.starts_with` have labelled parameters. Labels are useful when a function has several parameters of the same type; naming arguments allows to guess their purpose. Above, `~prefix:"state"` indicates `"less"` is passed as labelled argument `suffix`.
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
```suggestion
Some functions, such as `String.starts_with` have labelled parameters. Labels are useful when a function has several parameters of the same type; naming arguments allows to guess their purpose. Above, `~prefix:"state"` indicates `"state"` is passed as labelled argument `prefix`.
```

Sorry, my mistake here.

File: data/tutorials/language/0it_00_values_functions.md [hunk 3]
```diff
@@ -297,6 +297,25 @@ The `max` function returns the largest of its two arguments, which are:
...
+val cat : ?init:string -> string list -> string = <fun>
+
+# cat ["fri"; "sat"; "sun"];;
+- : string = "frisatsun"
+
+# cat ["fri"; "sat"; "sun"] ~init:"thr";;
+- : string = "thrfrisatsun"
+```
+
+Labelled parameters can have default values. This allows calling the function without specifying some arguments. Optional parameters when a function has many parameters or parameters which define some non nominal behaviour. Optional parameters are defined using a question mark `?` and passed using a tilde `~`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
```suggestion
Labelled parameters can have default values. This allows calling the function without specifying some arguments. Optional parameters are useful when a function has many parameters or parameters which often don't need to be specified. Optional parameters are defined using a question mark `?` and passed using a tilde `~`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]

**Comment by cuihtlauac** [comment]
@christinerose can you double check the changed text?

File: data/tutorials/language/0it_00_values_functions.md [hunk 4]
```diff
@@ -297,6 +297,25 @@ The `max` function returns the largest of its two arguments, which are:
...
+val cat : ?init:string -> string list -> string = <fun>
+
+# cat ["fri"; "sat"; "sun"];;
+- : string = "frisatsun"
+
+# cat ["fri"; "sat"; "sun"] ~init:"thr";;
+- : string = "thrfrisatsun"
+```
+
+Labelled parameters can have default values. This allows calling the function without specifying some arguments. Optional parameters are useful when a function has many parameters or parameters which often don't need to be specified. Optional parameters are defined using a question mark `?` and passed using a tilde `~`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
```
Comment (cuihtlauac) [inline data/tutorials/language/0it_00_values_functions.md]:
```suggestion
Some functions, such as `String.starts_with` have labelled parameters. Labels are useful when a function has several parameters of the same type; naming arguments allows to guess their purpose. Above, `~prefix:"state"` indicates `"state"` is passed as labelled argument `prefix`.

Labelled and optional parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]

**Pull request merged**